### PR TITLE
Documentación para la configuración de la carga reactiva

### PR DESCRIPTION
**Pull request by polsala**
### Description:

Added new documentation for reactive charge load configuration.

### Tasks:

- [x] Add reactive charge documentation.
- [x] Translations

### Links:

* [(CAT) distri/telegestio.md/_configuracio de la carrega reactiva_](http://builds.gisce.net/powerp-docs/powerp_docu_for_new_field_to_set_reactive_load_configuration/ca/distri/telegestion/#configuracio-de-la-carrega-reactiva)
* [(ES) distri/telegestio.md/_configuracion de la carga reactiva_](http://builds.gisce.net/powerp-docs/powerp_docu_for_new_field_to_set_reactive_load_configuration/es/distri/telegestion/#configuracion-de-la-carga-reactiva)

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_for_new_field_to_set_reactive_load_configuration/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_for_new_field_to_set_reactive_load_configuration/

**Comment by ecarreras** [comment]
@polsala falta la imatge crec

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_for_new_field_to_set_reactive_load_configuration/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_for_new_field_to_set_reactive_load_configuration/

File: docs/distri/telegestion.md [hunk 1]
```diff
@@ -166,6 +166,28 @@ infraestructura.
...
 
+## Configuració de la càrrega reactiva
+
+Hi ha dos maneres de configurar com volem la càrrega reactiva:
+- Configuració individualitzada per polissa (prioritaria):
+ - **Generar sempre la càrrega reactiva (always):** sempre generarem càrrega reactiva.
+ - **Càrrega reactiva sempre 0 (zero):** la càrrega reactiva sera sempre 0
+ excepte per potències superiors a 15 KW.
+ - **Sense càrrega reactiva (never):** No generarà càrrega reactiva excepte per potències superiors a 15 KW.
+ - **Ús de la variable de configuració (Per defecte):** fara ús de la variable de
```
Comment (ecarreras) [inline docs/distri/telegestion.md]: La llista no es visualitza de forma correcta

Comment (polsala) [reply docs/distri/telegestion.md]: Sip ara ho estava arreglant, en principi ara s'hauria de visualitzar bé

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_docu_for_new_field_to_set_reactive_load_configuration/

**Pull request merged**